commerce-frontend-api/src/main/java/com/liferay/commerce/frontend/ClayTableAction.java

```diff
@@ -87,6 +92,7 @@ public void setSeparator(boolean separator) {
 	private String _href;
 	private String _icon;
 	private String _label;
+	private final String _onClick;
```

Comment:
the `final` here stands out to me as inconsistent with the other variables here. If there's a particular reason for why this one is `final` and the others aren't, totally fine -- it just bugged me too much to pass up saying something about :)

Reply:
seems like SF auto added it due to `SourceCheck:JavaVariableTypeCheck`